tests/FunctionalJavascript/SettingsFormTest.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_authentication\FunctionalJavascript;
+
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Element\TraversableElement;
+use Drupal\Core\Config\ImmutableConfig;
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\oe_authentication\Form\AuthenticationSettingsForm;
+
+/**
+ * Tests the settings form.
+ */
+class SettingsFormTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_authentication_test',
+  ];
+
+  /**
+   * Tests the two-factor authentication condition settings.
+   */
+  public function testTwoFactorAuthenticationConditionSettings(): void {
+    $role_id = $this->createRole(['administer authentication configuration']);
+    $this->drupalLogin($this->createUser(values: ['roles' => [$role_id]]));
+    $this->drupalGet('/admin/config/system/oe_authentication');
+
+    $assert_session = $this->assertSession();
+    $use_2fa_fieldset = $assert_session->elementExists('named', ['fieldset', 'Use two-factor authentication']);
```

Comment:
```suggestion
    $use_2fa_radios_fieldset = $assert_session->elementExists('named', ['fieldset', 'Use two-factor authentication']);
```
If you don't see the form, you can easily get confused if this is just for the radios or some kind of outer wrapper for everything related to 2fa.

Reply:
It's a test. Not important.